src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -141,12 +141,116 @@
                   Publish
                 </v-btn>
               </div>
+              <p
+                :style="{ whiteSpace: 'pre-line' }"
+                v-text="publishDisabledMessage"
+              />
             </template>
-            <p
-              :style="{ whiteSpace: 'pre-line' }"
-              v-text="publishDisabledMessage"
-            />
           </v-tooltip>
+          <v-menu
+            v-if="publishDandiset.version_validation_errors.length"
```

Comment:
```suggestion
            v-if="publishDandiset.version_validation_errors"
```
Is `.length` required?

Reply:
Yup. Empty arrays are "truthy" in JS :roll_eyes: :